srcpkgs/zlib-ng/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'zlib-ng'
+pkgname=zlib-ng
+version=2.0.2
+revision=1
+build_style=cmake
+configure_args="-DZLIB_COMPAT=ON"
+short_desc="Zlib replacement with optimizations for 'next generation' systems"
+maintainer="Artur Sinila <opensource@logarithmus.dev>"
+license="Zlib"
+homepage="https://github.com/zlib-ng/zlib-ng"
+distfiles="https://github.com/zlib-ng/zlib-ng/archive/${version}.tar.gz"
+checksum=dd37886f22ca6890e403ea6c1d60f36eab1d08d2f232a35f5b02126621149d28
+
+post_install() {
+	mkdir "${DESTDIR}/usr/lib/zlib-ng" "${DESTDIR}/usr/include/zlib-ng"
+	mv "${DESTDIR}/usr/lib/pkgconfig/zlib.pc" "${DESTDIR}/usr/lib/pkgconfig/zlib-ng.pc"
+	mv ${DESTDIR}/usr/lib/libz.* ${DESTDIR}/usr/lib/zlib-ng/
+	mv ${DESTDIR}/usr/include/*.h ${DESTDIR}/usr/include/zlib-ng/
```

Comment:
This is just plain wrong! `pkg-config --cflags zlib-ng` and `pkg-config --libs zlib-ng` ;)